backend/Controllers/UserController.cs

```diff
@@ -24,21 +27,10 @@ public async Task<IActionResult> GetByUserName([FromRoute] string userName)
         return Ok(new UserResponse { Id = matchingUser.Id, UserName = matchingUser.UserName!, });
     }
 
-    [HttpGet("GetUserId/{userId}")]
-    public async Task<User?> GetByUserId([FromRoute] string userId)
+    [HttpPost("/{userId}/update")]
+    public async Task<IActionResult> UpdateUser([FromRoute] string userId, UpdateUserRequest userRequest)
     {
-        var matchingUser = await _userManager.FindByIdAsync(userId);
-        if (matchingUser == null)
-        {
-            return null;
-        }
-        return matchingUser;
-    }
-
-    [HttpPost("UpdateUser")]
-    public async Task<IActionResult> UpdateUser(UpdateUserRequest userRequest)
-    {
-        User? user = await GetByUserId(userRequest.Id.ToString());
+        User? user = await _userManager.FindByIdAsync(userId);
```

Comment:
This is also a database call and will want to happen in the usedr service Natasha and Anum are working on implementing this method right now - please can you add a TODO here referencing their branch name and to say that once that's been merged in service method can be used

Reply:
Discussed with Natasha and Anum, comments left on lines 19 and 31 for them to implement user services.